File: examples/rust-game/components/worker.rs
// Tag component — the second leg of the `["Hunger","Worker"]` query in
// scripts/hunger.rs (the labelle-engine#742 HungerController shape, ported to
// the native family). A GAME-SHAPED zero-field `labelle::component!`
// declaration (no `use` lines — the assembler stages the prelude): its schema
// extracts into scripting_components.zig as an empty persistent struct.
labelle::component! {
    Worker {}
}

File: examples/go-game/components/worker.zig
//! Tag component — the second leg of the `["Hunger","Worker"]` query in
//! scripts/hunger.go (the labelle-engine#742 HungerController shape,
//! ported to the go native family). A zero-field persistent struct;
//! Zig-authored for the same reason as components/hunger.zig (go ships
//! no declare tool in v1 — see that file's header).
pub const Worker = struct {};
